systemtest/src/test/java/io/strimzi/systemtest/operators/CustomResourceStatusST.java

```diff
@@ -605,8 +605,9 @@ void assertKafkaTopicWrongMinInSyncReplicasStatus(String topicName, String inval
         assertThat(kafkaTopicStatus.getConditions().stream()
             .anyMatch(condition -> condition.getType().equals(NotReady.toString())), is(true));
         assertThat(kafkaTopicStatus.getConditions().stream()
-            .anyMatch(condition -> condition.getReason().equals("InvalidRequestException")), is(true));
+            .anyMatch(condition -> condition.getReason().equals("InvalidTopicException")), is(true));
```

Comment:
Technically this is a compatibility break.

Reply:
Ah. Ok. Anyway, I think this PR needs to be eventually rebased to https://github.com/strimzi/strimzi-kafka-operator/pull/5988